yt_dlp/extractor/nhk.py

```diff
@@ -265,11 +321,11 @@ def _real_extract(self, url):
             entries.append(self._extract_episode_info(
                 urljoin(url, episode_path), episode))
 
-        program_title = None
-        if entries:
-            program_title = entries[0].get('series')
+        html = self._download_webpage(url, program_id)
+        program_title = clean_html(get_element_by_class('p-programDetail__title', html))
+        program_description = clean_html(get_element_by_class('p-programDetail__text', html))
```

Comment:
are we making the extra request here only for the playlist description? or does it improve title extraction too?